Cargo.toml

```diff
@@ -10,25 +10,29 @@ keywords = ["no_std", "framework", "oauth", "ace"]
 categories = ["authentication", "data-structures", "embedded", "encoding", "no-std"]
 repository = "https://github.com/namib-project/dcaf-rs"
 exclude = [
-    ".gitlab/*", ".gitlab-ci.yml"
+    ".gitlab/*", ".gitlab-ci.yml", ".github/*"
 ]
 
 [features]
-default = ["std"]
+default = ["std", "openssl"]
 std = ["serde/std", "ciborium/std", "serde_bytes/std", "erased-serde/std", "derive_builder/std"]
 
 [dependencies]
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 ciborium = { version = "^0.2", default-features = false }
 ciborium-io = { version = "^0.2", default-features = false }
-coset = { version = "^0.3", default-features = false }
+coset = { version = "^0.3", default-features = false, git = "https://github.com/namib-project/coset.git", branch = "fix_zero_length_map" }#, path = "../coset" }
```

Comment:
Needs to be upstreamed before merging

Reply:
PR is now open: google/coset#98

Reply:
Resolving for now, since I don't want to hold off on merging this for too long. We should wait for this before creating a new release, though.